Dashboard/app/js/lib/controllers/survey-controllers.js

```diff
@@ -615,6 +615,10 @@ FLOW.surveyControl = Ember.ArrayController.create({
     }
   }.observes('FLOW.selectedControl.selectedSurveyGroup'),
 
+  populateAll: function () {
+    FLOW.store.find(FLOW.Survey);
```

Comment:
So you probably need to set the retrieved survey array to `content` property of this controller